pkg/cmd/release/create/create.go

```diff
@@ -235,7 +251,7 @@ func createRun(opts *CreateOptions) error {
 		// of local tag status.
 		// If a remote tag with the same name as specified exists already
 		// then a new tag will not be created so ignore local tag status.
-		if tagDescription != "" && !existingTag && opts.Target == "" {
+		if tagDescription != "" && !existingTag && opts.Target == "" && !remoteTagPresent {
```

Comment:
added `!remoteTagPresent` check to avoid a duplicate tag lookup API call if it was already done above

Reply:
This is close, but `remoteTagPresent` isn't a good value to check for whether an API lookup was made, because an API lookup _might_ have been made and the result that was saved in the variable was `false`.

Instead, I would just perform this check in the case `--verify-tag` wasn't passed, because in the case of `--verify-tag` we already perform the server-side check:
```suggestion
		if tagDescription != "" && !existingTag && opts.Target == "" && !opts.VerifyTag {
```

Reply:
Updated in https://github.com/cli/cli/pull/6632/commits/fad72b788de367ee2ec6a54eccafb51250121c50